### PR TITLE
Add a clarification that this scenario assumes that a virtual network is already in place.

### PR DESCRIPTION
**Pull request by asukumar22**
Added a note stating that the scenario assumes a virtual network is already in place and added a link to the document to create a virtual network using Powershell if it is not already in place.

### CODE REVIEW

**Comment by Court72** [comment]
@EdPrice-MSFT 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

**Review by EdPrice-MSFT — APPROVED** [review]
Great. Thanks!

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Comment by PRMerger-2** [comment]
@EdPrice-MSFT: I'm sorry - only the author of this article, @, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: docs/reference-architectures/apis/protect-apis-content.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ This article doesn't address the application's underlying services, like App Ser
 
 ## Deploy this scenario 
 
+Note: This scenario assumes you already have a virtual network in place. If you need help creating a virtual network, see [Create a virtual network using PowerShell](/azure/virtual-network/quick-create-powershell).
```
Comment (Court72) [inline docs/reference-architectures/apis/protect-apis-content.md:83]:
```suggestion
> [!NOTE]
> This scenario assumes you already have a virtual network in place. If you need help creating a virtual network, see [Create a virtual network using PowerShell](/azure/virtual-network/quick-create-powershell).
```

**Pull request merged**